### ISSUE TITLE
"NPO Type" field integration

### ISSUE THREAD

**Issue by SovereignAndrey** [AngelProtocolFinance/angelprotocol-web-app#1917]

**Comment by SovereignAndrey** [comment]
@jp-mariano Please make sure that @abbasali1994 has the needed backend endpoint modifications for this expanded endowment reg portal payload (ie. added `npoType` field and that the field is returnable in the markeplace endowments list query as well as the single endowment record query. I know you did some work around this to hold a religous org field, so we might be able to use that with some tweaks? Please advise, thanks! :)

**Comment by jp-mariano** [comment]
Hey guys, @SovereignAndrey & @abbasali1994. The Charity Registration piece still needs a bit of work. Just to confirm, will this new `NPOType` field be included in the Documentation step? The current endpoints can still be used.

For the Endowment Edit Profile process, there already exists a field called `endow_designation` which just contains the "Non-Profit" and "Religious Non-Profit" values. We can repurpose this field to hold those values stated in the description instead.

**Comment by SovereignAndrey** [comment]
> Hey guys, @SovereignAndrey & @abbasali1994. The Charity Registration piece still needs a bit of work. Just to confirm, will this new `NPOType` field be included in the Documentation step? The current endpoints can still be used.

Yes, that's where we would have it as part of tier 1 initial data. 

> For the Endowment Edit Profile process, there already exists a field called `endow_designation` which just contains the "Non-Profit" and "Religious Non-Profit" values. We can repurpose this field to hold those values stated in the description instead.

Perfect! Let's go with that field naming then and sync the possible options we can hold there with the above starting list from Chauncey. 👍

**Comment by jp-mariano** [comment]
> Yes, that's where we would have it as part of tier 1 initial data.

Thanks for confirming.

> Perfect! Let's go with that field naming then and sync the possible options we can hold there with the above starting list from Chauncey. 👍

Alright, we can do with `EndowDesignation` for the Charity Registration piece instead.

**Comment by SovereignAndrey** [comment]
> Alright, we can do with `EndowDesignation` for the Charity Registration piece instead.

@jp-mariano So it would only be a matter of adding and `EndowDesignation` field to the intake for the EditProfile message and to the outbound EndowmentList & Endowment queries returned?

**Comment by jp-mariano** [comment]
> @jp-mariano So it would only be a matter of adding and EndowDesignation field to the intake for the EditProfile message and to the outbound EndowmentList & Endowment queries returned?

Yes, but we need to use `endow_designation` (because of the naming convention in the Endowment Profiles), it can be edited via adding that field into the EditProfile message. To include it in the return fields just add it in the `return` param for the EndowmentList query (I believe this is for the Marketplace). For a single Endowment query, I forgot, we also need to update that in order for it to return the correct field. Once it's updated, no changes will be required into the web app side.
